docs/src/components/Palette.tsx

```diff
@@ -0,0 +1,282 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import convertHexToRgba from "@kiwicom/orbit-design-tokens/lib/convertHexToRgba.js";
```

Comment:
I had to specify it because the current eslint rules expect all imports within docs to be tsx. I figured this one exception wasn't worth changing the rules for, but without the extension, the workflow fails.